mount/lookuptest/lookup_linux_test.go

```diff
@@ -0,0 +1,48 @@
+// +build linux
+
+// FIXME: we can't put this test to the mount package:
```

Comment:
Can this go in `testutil` package for now then? It is primarily testing `testutil.NewTestDevice`